benefit/data/sheets/CaseType/CaseType.json

```diff
@@ -1 +1 @@
-[{"LiveFrom": "01/01/2018", "ID": "Benefit", "Name": "SSCS Case v5.10.0${CCD_DEF_E}", "Description": "SSCS Benefit Cases", "JurisdictionID": "SSCS", "PrintableDocumentsUrl": "http://ccd-data-store-api-prod.service.core-compute-prod.internal/callback/jurisdictions/SSCS/case-types/Benefit/documents", "RetriesTimeoutURLPrintEvent": "5,10,15", "SecurityClassification": "PUBLIC"}]
\ No newline at end of file
+[{"LiveFrom": "01/01/2018", "ID": "Benefit", "Name": "SSCS Case v5.10.1${CCD_DEF_E}", "Description": "SSCS Benefit Cases", "JurisdictionID": "SSCS", "PrintableDocumentsUrl": "http://ccd-data-store-api-prod.service.core-compute-prod.internal/callback/jurisdictions/SSCS/case-types/Benefit/documents", "RetriesTimeoutURLPrintEvent": "5,10,15", "SecurityClassification": "PUBLIC"}]
```

Comment:
same comment about the version

Reply:
Thanks, as per git history, we increment with 1